final_task/rss_reader/string_operations.py

```diff
@@ -0,0 +1,64 @@
+
+def cut_string_to_length_with_space(basic_str, length):
+    """function that cut given string to the list of strings with a given max length"""
+    my_strings = []
```

Comment:
please avoid names such as `my...` in the production code

Reply:
Fixed